src/openrct2/ride/VehicleSubpositionData.cpp

```diff
@@ -24060,7 +24060,8 @@ static constexpr const rct_vehicle_info_list *TrackVehicleInfoListDefault[] = {
     &TrackVehicleInfo_96C062, &TrackVehicleInfo_96C3C4, &TrackVehicleInfo_96AC16, &TrackVehicleInfo_96AF78, &TrackVehicleInfo_96B2DA, &TrackVehicleInfo_96B63C,
     &TrackVehicleInfo_96C726, &TrackVehicleInfo_96CA88, &TrackVehicleInfo_96CDEA, &TrackVehicleInfo_96D14C, &TrackVehicleInfo_9761F2, &TrackVehicleInfo_9766C5,
     &TrackVehicleInfo_976B98, &TrackVehicleInfo_97706B, &TrackVehicleInfo_97753E, &TrackVehicleInfo_977A11, &TrackVehicleInfo_977EE4, &TrackVehicleInfo_9783B7,
-    &TrackVehicleInfo_97888A, &TrackVehicleInfo_978D5D, &TrackVehicleInfo_979230, &TrackVehicleInfo_979703,
+    &TrackVehicleInfo_97888A, &TrackVehicleInfo_978D5D, &TrackVehicleInfo_979230, &TrackVehicleInfo_979703, &TrackVehicleInfo_8BE57A, &TrackVehicleInfo_8BE69C,
```

Comment:
Where did you get the missing data from? And why did you (have to) add four entries?

Reply:
This is the subposition data. Each entry represents one angle of vehicle positions of a track piece, in order of TrackElemType then direction. So TrackVehicleInfoListDefault[0] is TrackElemType::Flat subposition data direction 0, TrackVehicleInfoListDefault[1] is TrackElemType::Flat subposition data direction 1. The four entries I added are, in order, TrackElemType::Flat subposition data direction 0, TrackElemType::Flat subposition data direction 1, TrackElemType::Flat subposition data direction 2, TrackElemType::Flat subposition data direction 3.

Reply:
I had to add four entries otherwise track element type 256 has no subposition data

Reply:
I see. Did you copy the subposition from an existing piece?

Reply:
yes, I copied the existing TrackElemType::Flat subposition data for directions 0 thru 3. If you search the file for the variable names they appear in order at the very beginning of the array as I explained.

Reply:
Interestingly enough, the value stored in track direction references this value without any other modifications.